README.md

```diff
@@ -69,6 +79,19 @@ For ADAL authentication, AZURE_CLIENT_ID and AZURE_CLIENT_SECRET also must be se
 
 5. Run ```mvn clean compile exec:java``` for a sample run through.
 
+## Use latest key vault SDK
+
+The key vault SDK package  here is **com.microsoft.azure.azure-keyvault**, if you are using the [latest](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets) version of the key vault SDK package, please reference to the following examples:
```

Comment:
```suggestion
The Key Vault secrets SDK here is **com.azure.azure-security-keyvault-secrets**, if you are using the [latest](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets) version of the key vault SDK package, please refer to the following examples:
```

Reply:
Key Vault is broken up into 3 different packages:
* https://search.maven.org/artifact/com.azure/azure-security-keyvault-certificates
* https://search.maven.org/artifact/com.azure/azure-security-keyvault-keys
* https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets